Dockerfile

```
@@ -67,7 +67,7 @@ WORKDIR /workspace/
 RUN git clone https://github.com/NVIDIA/Megatron-LM.git && \
   cd Megatron-LM && \
   git checkout 36e9b6bf3d8034b10c9bbd9fc357c2df2bd1515c && \
-  git cherry-pick -n e69187bc3679ea5841030a165d587bb48b56ee77 && \
+  git cherry-pick -n 02871b4df8c69fac687ab6676c4246e936ce92d0 && \
```

We should either cherry-pick both https://github.com/NVIDIA/Megatron-LM/commit/e69187bc3679ea5841030a165d587bb48b56ee77 and https://github.com/NVIDIA/Megatron-LM/commit/02871b4df8c69fac687ab6676c4246e936ce92d0, or we should just checkout https://github.com/NVIDIA/Megatron-LM/commit/02871b4df8c69fac687ab6676c4246e936ce92d0 directly:
```suggestion
  git checkout 02871b4df8c69fac687ab6676c4246e936ce92d0 && \
```

---

You are right, sorry about that, fixed it.